elastic/fleet-server
# filtering out UPDATE_TAGS and FORCE_UNENROLL

- Enhancement

## What is the problem this PR solves?

Avoid unnecessary traffic of delivering actions to agent.

## How does this PR solve the problem?

Ignore UI actions: `UPDATE_TAGS` and `FORCE_UNENROLL`, that don't need to be handled by agents.

## How to test this PR locally

- Start Fleet Server, enroll an Agent.
- Action Update tags and force unenroll from Fleet UI.
- Verify that these actions are not delivered to agents on check-in ("Action delivered" log is not present)

## Design Checklist


## Checklist



## Related issues

- Closes https://github.com/elastic/fleet-server/issues/2199
